modules/axhal/src/platform/aarch64_phytium_pi/mem.rs

```diff
@@ -0,0 +1,37 @@
+use crate::mem::*;
+use page_table_entry::{aarch64::A64PTE, GenericPTE, MappingFlags};
+
+/// Returns platform-specific memory regions.
+pub(crate) fn platform_regions() -> impl Iterator<Item = MemRegion> {
+    core::iter::once(MemRegion {
```

Comment:
Spintable is used for multi-core booting on Raspberry Pi, does it also used on Phytium Pi?